API.md

```diff
@@ -153,10 +165,14 @@ refresh_count and max_refresh_count can be used to determine when a re-authentic
 
 ### Authenticate with Cerberus as an App [POST]
 
-This endpoint takes IAM ARN information and generates an base 64 encoded KMS encrypted payload of the below. The ARN if registered with an SDB will have kms decrypt permissions on the KMS key that the payload was enrypted with.
+This endpoint takes IAM ARN information and generates a base64 encoded KMS encrypted payload. 
```

Comment:
The response object from this request is as follows

    {
      "auth_data": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
    }

The value of "auth_data" is the kms encrypted base64 encoded payload that has a structure of 

    {
       "client_token" : "e5fd901b-bc65-71e7-a214-6066fef1e918",
       "policies" : [ "foo-bar-read", "lookup-self" ],
       "metadata" : {
         "aws_region" : "us-west-2",
         "iam_principal_arn" : "arn:aws:iam::111111111:role/fake-role"
         "username" : "arn:aws:iam::111111111:role/fake-role"
         "is_admin": "false",
         "groups": "registered-iam-principals"
       },
       "lease_duration" : 3600,
       "renewable" : true
     }

I feel like that could be documented better here and was always a point of confusion for people trying to make new clients or do things manually.

Reply:
There were two examples that were different from each other.  I updated them to be the same and used the better example.